alluxio-python-library/alluxio/alluxio_file_system.py

```diff
@@ -278,6 +393,40 @@ def _create_session(self, concurrency):
         session.mount("http://", adapter)
         return session
 
+    def _load_file(self, worker_host, path, timeout):
+        try:
+            response = self.session.get(
+                self.LOAD_SUBMIT_URL_FORMAT.format(
+                    worker_host=worker_host,
+                    http_port=self.http_port,
+                    path=path,
+                ),
+            )
+            response.raise_for_status()
+            i = 0
+            while timeout is None or i <= timeout:
+                response = self.session.get(
+                    self.LOAD_PROGRESS_URL_FORMAT.format(
+                        worker_host=worker_host,
+                        http_port=self.http_port,
+                        path=path,
+                    ),
+                )
+                response.raise_for_status()
+                content = response.content
+                if b"SUCCEEDED" in content:
+                    logging.info(content.decode("utf-8"))
+                    return True
+                if b"FAILED" in content:
+                    raise Exception(f"{content}")
+                time.sleep(10)
+                i += 10
```

Comment:
when users provide timeout, they expect it as the total time to execute this function. In your case, the timeout < actual time taken = rounds * (10s + RPC round trip time)